.gitignore

```diff
@@ -0,0 +1,16 @@
+*.SAT
+*.obj
+*.glb
+*.sat
+*.STEP
+__pycache__
+*.txt
+*.tscn
+*.csv
+*.stl
+# python scripts used for testing purposes
+stl_to_obj/bpy_test.py
```

Comment:
All the file extensions should be a good generalization for all users, because they are large input/output files that we don't want in the repo. As for the extra python scripts and json files that I've included, we can either continue with how it is, or bundle them all into one folder (which just pushes off the problem even further since that entire folder won't be committed into the repo).

Reply:
@gonuke this `bpy_test.py` file is more a "sandbox" than a testing suite. I think for now this is fine, but maybe by next PR it's probably time to move beyond sandboxing and switching to something like `pytest`?

Reply:
Just cleaned up the `.gitignore` and removed all the personal files. I'm now using `.git/info/exclude" to ignore those files.